src/nf.js

```diff
@@ -6,8 +6,21 @@ export class NfValueConverter {
     this.service = i18n;
   }
 
-  toView(value, formatOptions, locale, numberFormat) {
-    let nf = numberFormat || this.service.nf(formatOptions, locale || this.service.getLocale());
+  toView(value, nfOrOptions, locale, nf) {
+    if (value === null
+      || typeof value === 'undefined'
+      || (typeof value === 'string' && value.trim() === '')
+      ) {
+      return value;
+    }
+
+    if (nfOrOptions && (typeof nfOrOptions.format === 'function')) {
+      return nfOrOptions.format(value);
+    } else if (nf) {
+      console.warn('This ValueConverter signature is depcrecated and will be removed in future releases. Please use the signature [nfOrOptions, locale]'); // eslint-disable-line no-console
```

Comment:
I've simply copied the date time value converter code which was implemented by an aurelia developer. I even questioned how they wanted it done [here](https://github.com/aurelia/i18n/issues/100#issuecomment-213424746).

I propose if this needs changing it is done in a separate PR and both are updated.